### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by acarvalh**

### CODE REVIEW

File: NonResonant/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+In "Hist2DSum_V0_SM_box.root" you find the distribution for both 
+     - the SM point (simulated from 100k events) - H0binX  
+     - sum of the events simulated to Mriond 2016 (SM + box + the V0 benchmarks from 2-13 (simulated from ~300 events/each) - H1binX
```
Comment (OlivierBondu) [inline NonResonant/README.md]: The bullet list shows broken as it is not following the [markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)
also: 
  * Mriond -> Moriond
  * 300 events -> 300k events

File: NonResonant/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+where X = 1 means the binning ( 90,0.,1800.,10,-1,1. ) and X = 1 means the same binning used for the analytical formula (see bellow)
+
+     binGenMHH = [250.,270.,300.,330.,360.,390., 420.,450.,500.,550.,600.,700.,800.,1000.] 
+     binGenCostS  = [ -1., -0.55,0.55,1.  ] 
+ 
+==================================================================================================================
+In "Distros_5p_SM3M_sumBenchJHEP_13TeV.root" you find the distribution for both 
+     - the SM point (simulated from 3M events) - H0binX  
+     - sum of the events from simulation of the 12 benchmarks defined in 1507.02245v4 (JHEP version) - H1binX
```
Comment (OlivierBondu) [inline NonResonant/README.md]: Could you add the actual link to the arxiv ?

File: NonResonant/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+     binGenMHH = [250.,270.,300.,330.,360.,390., 420.,450.,500.,550.,600.,700.,800.,1000.] 
+     binGenCostS  = [ -1., -0.55,0.55,1.  ] 
+ 
+==================================================================================================================
+In "Distros_5p_SM3M_sumBenchJHEP_13TeV.root" you find the distribution for both 
+     - the SM point (simulated from 3M events) - H0binX  
+     - sum of the events from simulation of the 12 benchmarks defined in 1507.02245v4 (JHEP version) - H1binX 
+
+===================================================================================================================
+In "Distros_5p_500000ev_12sam_13TeV_JHEP_500K.root" you find the 2D histograms for the 12 benchmarks defined in 1507.02245v4 (JHEP version) ( 90,0.,1800.,10,-1,1. )
```
Comment (OlivierBondu) [inline NonResonant/README.md]: Same here, please add the link

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
Thanks ! looking good, only minor changes on my end :)

thanks !

**Comment by acarvalh** [comment]
done

2017-01-06 10:10 GMT+01:00 Olivier Bondu <notifications@github.com>:

**Review by OlivierBondu — APPROVED** [review]

**Pull request merged**